### PR TITLE
fix #75915 videokontroldoma.ru

### LINKED ISSUE #75915: videokontroldoma.ru
### Issue URL (Ads)
 [https://videokontroldoma.ru/podklyuchenie-trubki-domofona/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fvideokontroldoma.ru%2Fpodklyuchenie-trubki-domofona%2F)
### Comment
Username: `@spok1`
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#75915

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
This is the ad leftover, so please make a rule for the russian specific

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -757,7 +757,8 @@ toster.ru###register-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27068
 ||montpellierhandball.com/scripts/fancybox/minified__fancybox.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26451
-arcadepunks.com###wpfront-notification-bar
+! https://github.com/AdguardTeam/AdguardFilters/issues/75915
+videokontroldoma.ru,arcadepunks.com###wpfront-notification-bar
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/annoyances.txt:761]: This is the ad leftover, so please make a rule for the russian specific

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt]: done

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -757,6 +757,7 @@ toster.ru###register-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27068
 ||montpellierhandball.com/scripts/fancybox/minified__fancybox.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26451
+! https://github.com/AdguardTeam/AdguardFilters/issues/75915
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/annoyances.txt:760]: @wout-codes please remove this comment

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Pull request merged**